crates/weaver_cache/Cargo.toml

```diff
@@ -14,7 +14,7 @@ workspace = true
 [dependencies]
 tempdir = "0.3.7"
 dirs = "5.0.1"
-gix = { version = "0.61.1", default-features = false, features = [
+gix = { version = "0.61.0", default-features = false, features = [
```

Comment:
We should do these bumps w/ dependabot PRs I think.

Reply:
I agree